TwilioVerifySDK/TwilioSecurity/Sources/Storage/SecureStorage.swift

```diff
@@ -21,37 +21,44 @@ import Foundation
 
 ///:nodoc:
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/9cbd540c6135af3d1641ad17aef5ccc5e04d146c/TwilioVerifySDK/TwilioSecurity/Sources/Storage/SecureStorage.swift#L22"></span>Prefer at least one space after slashes for comments.<br />
<code>comment_spacing</code> <code>SecureStorage.swift:22</code></td>
    </tr>
  </tbody>
</table>